src/Psr6Store.php

```diff
@@ -98,6 +100,7 @@ public function __construct(array $options = [])
         $this->options = $resolver->resolve($options);
         $this->cache = $this->options['cache'];
         $this->lockFactory = $this->options['lock_factory'];
+        $this->hashAlgorithm = version_compare(PHP_VERSION, "8.1.0", ">=") ? 'xxh128' : 'sha256';
```

Comment:
@Toflar should we make it configurable over an option? and use this in setDefault?

Reply:
I don't think there's much value to having it configurable. You'd be limited to the hash functions PHP offers and I think that the community would be able to agree on the best fit for this cache.
If we wanted to have anything configurable, we should probably rather have a `protected function hash...` which one can override. But this too isn't necessary imho. Nobody asked for it in all these years ;-)